library/std/src/sys/itron/abi.rs

```diff
@@ -0,0 +1,176 @@
+//! ABI for μITRON derivatives
+pub type int_t = crate::os::raw::c_int;
+pub type uint_t = crate::os::raw::c_uint;
+pub type bool_t = int_t;
+
+/// Kernel object ID
```

Comment:
Almost everything in `sys` is `pub` right now though; the module itself is private.

Reply:
> Almost everything in sys is pub right now though; the module itself is private.

Yeah, the reason to use `crate` visibility, especially for _new_ code is because it makes reviewing easier and visibility mistakes very unlikely to impossible.